netket/graph/lattice.py

```diff
@@ -296,10 +223,29 @@ def __init__(
               [1.         0.8660254 ]
               [0.75       1.29903811]
               [1.25       1.29903811]]
-        """
-        if max_neighbor_order < 1:
-            raise ValueError("max_neighbor_order must be >= 1.")
 
+            Constructs a rectangular lattice with distinct horizontal and vertical edges:
+
```

Comment:
Can you check that this example renders correctly in the documentation?

Reply:
How do I do that?

Reply:
in a shell
```
cd my_netket_folder
pip install -r docs/requirements.txt
cd docs
make html
open _build/html/index.html
```

Reply:
Yeah, it looks as it should

Reply:
thanks for checking!